.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+patches/ @bridiver
+chromium_src/ @bridiver
+script/apply-patches.py @bridiver
```

Comment:
I'm not really concerned about this one, just the actual patches

Reply:
updated 👍